olm-catalog/serverless-operator/project.yaml

```diff
@@ -64,9 +64,9 @@ dependencies:
   operator: 1.6.0
   # Previous versions required for downgrade testing
   previous:
-    serving: 1.7.0
-    eventing: 1.7
-    eventing_kafka_broker: 1.7
+    serving: knative-v1.8
```

Comment:
should these be `1.8.0` ? 
(by looking at the prev. two PRs for this exercise?)

Reply:
I think Serving is on the same pattern from 1.8 https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-v1.8__412.yaml#L166-L170.

Reply:
Yes, we are on the same pattern as Eventing now. Should be fine here too.